monkeytype/cli.py

```diff
@@ -147,9 +151,10 @@ def apply_stub_handler(args: argparse.Namespace, stdout: IO, stderr: IO) -> None
             src_path
         ])
         try:
-            subprocess.run(cmd, shell=True, check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+            proc_output = subprocess.run(cmd, shell=True, check=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+            print(proc_output.stdout.decode('utf-8'), file=stdout)
         except subprocess.CalledProcessError as cpe:
-            raise HandlerError(f"Failed applying stub with retype:\n{cpe.stdout.decode('utf-8')}")
+            raise HandlerError(f"Failed applying stub with retype:\n{cpe.stderr.decode('utf-8')}")
```

Comment:
I think this change is wrong, when we pass `stderr=subprocess.STDOUT` to the `subprocess.run` call, we are telling it to redirect stderr to stdout, so it all gets combined on stdout, we will never get anything on stderr. Did you see something specific that made you think this change was needed?

Reply:
I tested both ways and also tested the subprocess.run redirection several times.  This was the combo that gave me both warning and error outputs, but it's possible I mixed it up.  I'll retest it.

Reply:
Reverted and tested, still get the warning.